discogs_importer.user.js

```diff
@@ -386,6 +386,29 @@ function parseDiscogsRelease(data) {
         // Track position and release number
         var trackPosition = discogsTrack.position;
 
+        // Handle sub-tracks
+        if (trackPosition == "" && discogsTrack.sub_tracks) {
+            trackPosition = discogsTrack.sub_tracks[0].position;
```

Comment:
Note: track position isn't correct in all cases,  ie. http://www.discogs.com/release/1156598 will have track B in release editor set as track 5. I found various weird formats in positions like 'AAa' or 'Ab' or 'AA1'... the current script doesn't handle those.